app/src/renderer/vuex/modules/send.js

```diff
@@ -124,7 +124,7 @@ export default ({ node }) => {
       } else {
         // get private key to sign
         const wallet = state.externals.getKey(
-          rootState.user.account,
+          rootState.session.localKeyName,
```

Comment:
i think i do prefer `localKeyPairName` as it cannot be confused as an object key and implies that a private key is involved which is important.